doc/design/garbage-collection-assets.md

```diff
@@ -0,0 +1,40 @@
+# Asset Garbage Collection
+
+## Background
+
+Now that the [design for asset blob and upload garbage collection](./garbage-collection-uploads-asset-blobs-2.md) is deployed to staging, we are ready to implement Asset garbage collection. The garbage collection process in this design document only applies to “regular” assets; garbage collection of zarrs is not covered and is out of scope for this design doc.
+
+## What is an “Orphaned Asset”
+
+It’s important to precisely define what it means for an asset to be eligible for garbage collection (an “orphaned asset”).
+An orphaned asset has the following properties:
+- The asset is not currently associated with any Versions
+- The asset is older than 30 days
```

Comment:
```suggestion
- The asset was not modified for more than 30 days
```

or

```suggestion
- The asset is older than 30 days based on `modified` datetime-stamp
```
as opposed to `created`, since we do have operations which modify assets in place and `modified` does differ from `created`